DS4Windows/DS4Forms/MainWindow.xaml.cs

```diff
@@ -1288,7 +1288,20 @@ private void NewProfBtn_Click(object sender, RoutedEventArgs e)
         // Ex Mode Re-Enable
         private async void HideDS4ContCk_Click(object sender, RoutedEventArgs e)
         {
-            if (DS4Windows.Global.UseExclusiveMode == true) { MessageBox.Show("This feature is depreciated and no longer supported. Exclusive mode usage is provided mearly as a legacy feature. Do NOT ask for help for this feature, you will not recieve any.", "Feature no longer supported"); }
+            if (DS4Windows.Global.UseExclusiveMode == true && !Properties.Settings.Default.ContCk_Understood)
+            {
+                MessageBoxResult result = MessageBox.Show
+                (
+                    "This feature is deprecated and no longer supported. Exclusive mode usage is provided mearly as a legacy feature. Do NOT ask for help for this feature, you will not recieve any.\n\nDo you understand?",
+                    "Deprecation Warning",
+                    MessageBoxButton.YesNo,
+                    MessageBoxImage.Warning,
+                    MessageBoxResult.None
+                );
+
+                Properties.Settings.Default.ContCk_Understood = result == MessageBoxResult.Yes;
```

Comment:
Could maybe return if the user did not understand, we don't want them using a feature, expecting support if they don't know what the feature is or they don't understand the deprecation warning.

```suggestion
                if (result != MessageBoxResult.Yes) return;

                Properties.Settings.Default.ContCk_Understood = true;
```